src/constant/index.ts

```diff
@@ -0,0 +1,9 @@
+export const Register = {
+  TEXT_STEP1: "간편 회원가입을 위해<br /> 약관 동의가 필요해요",
+  TEXT_STEP2: " 학교 계정 인증을 위한<br /> 이메일을 입력하세요",
+  DESCRIPTION_STEP2: " 최초 1회 이메일 인증이 필요해요.",
+  TEXT_STEP3: "나만의 프로필을<br /> 설정하세요",
+  DESCRIPTION_STEP3: "마이페이지에서 언제든 수정 가능해요.",
+  TEXT_STEP4: "추가적인 정보를<br /> 입력하세요",
+  DESCRIPTION_STEP4: "추가정보를 입력하고 신뢰도를 높여보세요!",
+};
```

Comment:
이런 enum 타입같은 경우는 `as const`를 사용하면 좋아요!!
`as const`를 사용하게 되면  리터럴 타입으로 정확하게 추론되는 장점이 있고, 객체의 모든 프로퍼티가 readonly 프로퍼티가 되어 실수로 값이 변경되는 것을 방지할 수 있다는 장점도 있어요!!


```suggestion
export const Register = {
  TEXT_STEP1: "간편 회원가입을 위해<br /> 약관 동의가 필요해요",
  TEXT_STEP2: " 학교 계정 인증을 위한<br /> 이메일을 입력하세요",
  DESCRIPTION_STEP2: " 최초 1회 이메일 인증이 필요해요.",
  TEXT_STEP3: "나만의 프로필을<br /> 설정하세요",
  DESCRIPTION_STEP3: "마이페이지에서 언제든 수정 가능해요.",
  TEXT_STEP4: "추가적인 정보를<br /> 입력하세요",
  DESCRIPTION_STEP4: "추가정보를 입력하고 신뢰도를 높여보세요!",
} as const;
```